README.md

```diff
@@ -868,6 +868,7 @@ Table of Contents
   * [Divio](https://www.divio.com/) - A platform to manage cloud application deploying only using Docker. Available free subscription for development projects.
   * [Koyeb](https://www.koyeb.com) - Koyeb is a developer-friendly serverless platform to deploy apps globally. Seamlessly run Docker containers, web apps, and APIs with git-based deployment, native autoscaling, a global edge network, and built-in service mesh and discovery. Koyeb provides two nano services to run your apps with its forever-free tier and also sponsors open-source projects with free resources.
   * [Railway](https://railway.app) - Railway is an infrastructure platform where you can provision infrastructure, develop with that infrastructure locally, and then deploy to the cloud. 3 Projects, 2 Plugins / Project, 2 Environments / Project, 3 Live Deploys / Environment available for free.
+  * [gupshup.io](https://www.gupshup.io/) - A Bot Platform to develop, test and deploy Conversational Messaging Bots for free on multiple Social Media channels.
```

Comment:
I can see it says "start for free" but cant see details of pricing or whats free, do you have a link?

Reply:
Currently, only bots published on Whatsapp are charged as Whatsapp's Business API is chargeable. The pricing for the same can be found [here](https://www.gupshup.io/developer/whatsapp-api/#whatIsPricing). Publishing on all other channels like FB, Telegram, Viber, Slack, Twitter, etc. is free which can be verified by logging in and publishing a bot.

Reply:
OK great, afraid though without seeing this on the site I cannot add it really. You'll appreciate with like 1000 services here if I had to signup and do something like "publish a bot" to verify the free claim the list would never exist :)

Reply:
No Issues. Thanks. I'll update once concrete pricing details are put up on the website.